app/presenters/brexit_landing_page_presenter.rb

```diff
@@ -12,7 +12,6 @@ class BrexitLandingPagePresenter
 
   def initialize(taxon)
     @taxon = taxon
-    @comms = fetch_comms
```

Comment:
We can remove the `:comms` attr_reader now

Reply:
Good spot! Updated.